iree/turbine/kernel/compiler/ir.py

```diff
@@ -46,7 +46,7 @@
     llvm as llvm_d,
     math as math_d,
     memref as memref_d,
-    rocdl as rocld_d,
+    rocdl as rocdl_d,
```

Comment:
> The actual imported symbol doesn't appear to ever get used, but seems better to have this right.

BTW for this, I'm considering wrapping this import in a `try: except:` to maintain some compatibility with prior versions of the IREE packages... at least the import is used.

Reply:
Yeah I ran into the same failure as someone was discussing on the discord, which is how I noticed this typo :-)